diagnostics/top_heaviness_metric/top_heaviness_ratio_robustness_calc.py

```diff
@@ -0,0 +1,179 @@
+# Last update: 30 May top_heaviness_ratio_robustness_calc.py
+import os
+import xarray as xr
+import numpy as np
+import scipy 
+from scipy import interpolate
+from scipy import integrate
+import matplotlib.pyplot as plt
+from mpl_toolkits.basemap import Basemap
```

Comment:
Basemap is no longer supported.  Please rewrite the plotting using another package, such as Cartopy.

Reply:
We have updated the code with the cartopy module.